content/source/docs/cloud/workspaces/run-tasks.html.md

```diff
@@ -5,93 +5,91 @@ page_title: "Run Tasks - Workspaces - Terraform Cloud and Terraform Enterprise"
+**Permissions** - In order to create a run task, you must have a user account that has been assigned the organization owner permission. You must be at least a workspace administrator in order to associate a run task to a given workspace.
 
 ## Configuring a Run Task
 
-1. Navigate to the desired workspace, then select "Tasks" from the "Settings" menu. A page appears where you can either create a new event hook or select an existing one that is not connected to the workspace.
+1. Navigate to the desired workspace, then click **Run Tasks** from the **Settings** menu.
 
-![Screenshot: a workspace's settings drop-down menu](./images/run-tasks-workspace-settings.png)
+    ![Screenshot: a workspace's settings drop-down menu](./images/run-tasks-workspace-settings.png)
 
-2. Click "Create a new event hook." The "Task event hooks" page appears. 
+2. Click **Create a new run task** which will take you to the **Run Tasks** integration page.
 
-3. Click "Create hook."
+3. Click **Create run task**
 
-![Screenshot: an organization's blank task event hooks state](./images/run-tasks-blank-event-hooks.png)
+    ![Screenshot: an organization's blank run tasks state](./images/run-tasks-empty-run-tasks.png)
 
-4. Enter the information about the event hook to be configured:
+4. Enter the information about the run task to be configured:
+ - **Name** (required): A human-readable name for the run task. This will be displayed in workspace configuration pages and can contain letters, numbers, dashes and underscores.
+ - **Endpoint URL** (required): The URL for the external service. Run tasks will POST the [run tasks payload](../integrations/run-tasks/index.html#integration-details) to this URL.
+ - **HMAC key** (optional): A secret key that may be required by the external service to verify request authenticity.
 
-- **Name** (required): A human-readable name for the event hook. This will be displayed in workspace configuration pages and can contain alphanumeric characters, dashes, and underscores.
-- **Hook endpoint URL** (required): The URL where your external service is listening for a [run tasks payload](../api/run-tasks.html).
-- **HMAC key** (optional): A key that your remote endpoint can use to verify that requests are originating from Terraform Cloud.
+5. After you have entered the necessary information, click **Create run task**.
 
-5. After you have entered the necessary information, click "Create event hook." A page with a list of configured event hooks appears.
+    ![Screenshot: an organization's populated run tasks state](./images/run-tasks-run-tasks-populated.png)
 
-![Screenshot: an organization's populated task event hooks state](./images/run-tasks-event-hooks-populated.png)
+6. Click **Workspaces** in the top navigation, click your workspace from the grid, and then click **Run Tasks** in the **Settings** menu.
```

Comment:
```suggestion
6. Click **Workspaces** in the top navigation bar, click your workspace in the grid, open the **Settings** menu and select **Run Tasks**.
```

Reply:
Should this be the start of a new section called "Adding Run Tasks to a Workspace"?  It just seems like we've jumped from creating it to a new thing that users may want to do.